src/main/java/racingcar/Application.java

```diff
@@ -1,10 +1,63 @@
 package racingcar;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
+import utils.RandomUtils;
 
 public class Application {
+
+    private static final int STANDARD_NUMBER = 4;
+
     public static void main(String[] args) {
         final Scanner scanner = new Scanner(System.in);
         // TODO 구현 진행
+        OutputView outputView = new OutputView();
+        InputView inputView = new InputView();
+        outputView.printMsg("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        Car[] cars = inputView.readCars(scanner);
+
+        outputView.printMsg("시도할 회수는 몇 회인가요?");
+        int racingCnt = inputView.readRacingCount(scanner);
+
+        outputView.printMsg("\n실행결과");
+        for (int x = 0; x < racingCnt; x++) {
+            for (int carIdx = 0; carIdx < cars.length; carIdx++) {
+                setCarPosition(cars, carIdx);
+            }
+            outputView.printCarPosition(cars);
+        }
+
+        List<Car> winnerCarsList = getWinnerCars(cars);
+        outputView.printWinner(winnerCarsList);
+    }
+
+
+    private static void setCarPosition(Car[] cars, int carIdx) {
+        int randomNum = RandomUtils.nextInt(0, 9);
+        if (randomNum > STANDARD_NUMBER) {
+            cars[carIdx].addPosition();
+        }
+    }
+
+    private static int getMaxPosition(Car[] cars) {
+        int maxNum = -1;
+        for (int i = 0; i < cars.length; i++) {
+            if (maxNum < cars[i].getPosition()) {
+                maxNum = cars[i].getPosition();
+            }
+        }
```

Comment:
for each문을 이용하면 length를 쓰지 않아도 됩니다.
```suggestion
       for (Car car : cars) {
            if (maxNum == car.getPosition()) {
                winnerCarsList.add(car);
            }
        }
```

Reply:
for each 문으로 사용하니 쓴 사람도 읽는사람도 읽기 쉬워지는 것 같아요.